File: Java/Array/ArrayPartitionI.java
import java.util.*;

public class Solution {
    public int arrayPairSum(int[] nums) {
        int result = 0;
        Arrays.sort(nums);
        for (int i = 0; i < nums.length; i += 2) {
            result += nums[i] < nums[i + 1] ? nums[i] : nums[i + 1];
        }
        return result;
    }
}

File: Python/Array/ArrayPartitionI.py
class Solution(object):
    def arrayPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        result = 0
        nums = sorted(nums)
        for i in range(0, len(nums), 2):
            result += min(nums[i], nums[i + 1])
        return result